### PR TITLE
Transform numeric to number because it is stored in the database that way

### PR DESCRIPTION
**Pull request by mathiashelin**
## Type

> remove the types that don't apply
- Enhancement

## Pull request description

In js, we'd like to use type 'number' as it is more correct than 'numeric'. We need to change this in formbuilder, because in the database the fields are stored as type 'numeric'.

### CODE REVIEW

**Comment by StijnVrolijk** [comment]
Why is 'number' more correct than 'numeric'?

**Comment by mathiashelin** [comment]
Because type="numeric" doesn't exist in html.

File: src/Frontend/Modules/FormBuilder/Widgets/Form.php [hunk 1]
```diff
@@ -322,6 +321,11 @@ private function loadForm()
     private function setCustomHTML5ErrorMessages(array $item, SpoonFormAttributes $formField)
     {
         foreach ($item['validations'] as $validation) {
+            // we need to change it here since it is saved like this in the database and we can't change that for now.
+            if ($validation['type'] === 'numeric') {
```
Comment (tijsverkoyen) [inline src/Frontend/Modules/FormBuilder/Widgets/Form.php]: Wouldn't it be more correct to store the correct value?

Comment (carakas) [inline src/Frontend/Modules/FormBuilder/Widgets/Form.php]: you can change that in the backend but it needs to be implemented in a backwards compatible way.

File: src/Frontend/Modules/FormBuilder/Widgets/Form.php [hunk 2]
```diff
@@ -322,6 +321,11 @@ private function loadForm()
     private function setCustomHTML5ErrorMessages(array $item, SpoonFormAttributes $formField)
     {
         foreach ($item['validations'] as $validation) {
+            // we need to change it here since it is saved like this in the database and we can't change that for now.
```
Comment (carakas) [inline src/Frontend/Modules/FormBuilder/Widgets/Form.php]: I would maybe add a @TODO here so we can find this when we prepare for fork 5

File: src/Frontend/Modules/FormBuilder/Widgets/Form.php (hunk 1 above)
Comment (tijsverkoyen) [inline src/Frontend/Modules/FormBuilder/Widgets/Form.php]: Well, it should.
Maybe we should also add a `@deprecated`-line, so this code can be removed.

**Pull request merged**